pkg/command/root.go

```diff
@@ -471,6 +472,18 @@ var opsCmd = &cobra.Command{
 	},
 }
 
+var tpeCmd = &cobra.Command{
+	Use:     "platform-engineering",
+	Short:   "Commands that provide functionality for Tanzu Platform Engineering",
+	Aliases: []string{"tpe"},
+	Annotations: map[string]string{
+		"group": string(plugin.ExtraCmdGroup),
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		handleTargetHelp(cmd, args)
```

Comment:
Since we now have this great way to add a group of commands without having to add a new target, could you update the printout that users can see in `handleTargetHelp()` to avoid mentioning "target"?
```
$ tz platform-engineering
Note: No plugins are currently installed for the "platform-engineering" target.

Commands that provide functionality for Tanzu Platform Engineering
[...]
```
Maybe we can say:
`Note: No installed plugin provides a sub-command for the  "platform-engineering" command`

Reply:
Thanks, because it is a something that only gets invoked when the command is hidden, I wasn't too concerned.
That said, I had similar thoughts and was thinking of renaming the function and doing away with the last word (target) altogether since over time there should be little distinction between a target and a plain command group since that's how we want the former to be viewed. wdyt?

Reply:
Sounds good to me.